fix(tag): default theme error #55

lockiechen opened the pull request:
affects: @bkui-vue/shared, @bkui-vue/tag

<!--
请保证PR标题明确清晰, 易于理解
Keep PR title verbose enough

相关issue可以是 #编号 或者贴 issue链接
`Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->


## 变更点(Changes)

- xxxx

## 相关issues (Which issues this PR fixes)

- Fixes #

## 备注(Special notes)

lockiechen committed:
fix(tag): default theme error #55

affects: @bkui-vue/shared, @bkui-vue/tag

tencentblueking-adm commented:
[![CLA assistant check](https://bk-cla.bktencent.com:443/pull/badge/signed)](https://bk-cla.bktencent.com:443/TencentBlueKing/bkui-vue3?pullRequest=763) <br/>All committers have signed the CLA.

lockiechen committed:
fix(icon): fix icon size #55

affects: @bkui-vue/icon

lockiechen committed:
fix(shared): type conflict #55

affects: @bkui-vue/alert, @bkui-vue/dialog, @bkui-vue/divider, @bkui-vue/dropdown, @bkui-vue/input,
@bkui-vue/modal, @bkui-vue/popover, @bkui-vue/select, @bkui-vue/shared, @bkui-vue/steps,
@bkui-vue/tag
